docs/assets/demo.ts

```diff
@@ -39,7 +49,7 @@ export const parentNodes: GraphNode[] = [
       return {
         id: `n-2-n-${i}`,
         label: `Node 2 > ${i} - 1 Child`,
-        parent: 'n-2',
+        parents: ['n-2'],
```

Comment:
Does the user have to specify this? If so we need to remove this and use the ngraph function for getting outgoing links - 

https://github.com/anvaka/ngraph.graph/blob/main/index.js#L174

```
const links = graph. getLinks(nodeId);
const incoming = links.filter(l => l.to === nodeId);
// incoming = parents
```

Reply:
do we want to apply this to all nodes (allowing for all nodes with links to be collapsible and then necessitating a property allowing collapsing or not)? the original intention by manually specifying it would be to allow only specified nodes to be collapsible

Reply:
also, by not specifying the parents, we would then need to generating edges for these collapsible children nodes (which is currently not needed with the parents field)

Reply:
so yeah, seems like this is a matter of creating nodes/edges like normal. and having a property allowing collapsing/expanding of nodes with outgoing links